### DISCUSSION TITLE
QTable v-slot:item is slow

### DISCUSSION THREAD

**Discussion by RamonDonadeu** [quasarframework/quasar#15457] [General - Components / Directives / etc]
So, I have a personalized table, when is in desktop mode is formed by rows, but when is for mobile devices it's in grid mode. 
Our team found out that when it's on grid mode, the table load and selection is very very slow, it has a lot of delay. But if we remove all of the "template v-slot:item> </> "It is a lot faster. 

This is standard table grid:
https://codepen.io/RamonDonadeu/pen/mdGJXNP?editors=1011

This is customized table grid:
https://codepen.io/RamonDonadeu/pen/MWqwQLV?editors=1011

If you select a row you can see how it has some delay when clicking. Obviously, it only happens when there are a lot of rows.

But if the structure is in normal mode it works fine:
https://codepen.io/RamonDonadeu/pen/ExejEpa?editors=1010

I don't know if there is any solution (Apart of using pagination) But anyway, maybe this is a bug? Or something is wrong because for me it doesn't makes any sense that it works well with normal structure and not with grid structure.

**Comment by metalsadman** [comment]
nothing's wrong, both are slow to me. like you said `it only happens when there are a lot of rows.`,

**Reply by RamonDonadeu** [reply]
But I feel, that the customized grid is slower than the other two, and has the same amount of rows.

The 1st and 3rd are toggled after realease click, and the 2nd takes like... 0.4 seconds or something like that. And for my complex tables it is a big problem. I just letting people know, because maybe there is room for improvement. :)

**Reply by metalsadman** [reply]
pagination/virtual scrolling/ server side pagination is what I would do in these cases if can't avoid not to make the rows reactive.
